.evergreen/config.yml

```diff
@@ -1123,7 +1123,23 @@ tasks:
       - func: install dependencies
       - func: bootstrap mongohoused
       - func: run data lake tests
-  - name: test-load-balancer
+  - name: test-load-balancer-6.0.0
```

Comment:
Can we flip the name to align with the other test tasks? (also use load-balanced to match the topo type in the driver)
- `test-5.0-load-balanced`
- `test-6.0-load-balanced`
- `test-latest-load-balanced`